docs/source/api.rst

```diff
@@ -0,0 +1,56 @@
+..
+.. Copyright (C) Arm Ltd. 2021.  ALL RIGHTS RESERVED.
+..
+.. See file LICENSE for terms.
+..
+
+API
+=============
+
+*  The full API specification can be found here : `HTML <https://openucx.github.io/ucx/api/v{VERSION}/html/index.html>`_ `PDF <https://openucx.github.io/ucx/api/v{VERSION}/ucx-v{VERSION}.pdf>`_
+* `Code examples <https://github.com/openucx/ucx/tree/v{VERSION}.x/examples>`_
+
+Application Context routines
+********************************
+.. doxygengroup:: UCP_CONTEXT
+    :path: build/xml
+
+Worker routines
+*******************
+.. doxygengroup:: UCP_WORKER
```

Comment:
can we build this token list automatically somehow?

Reply:
They do have an external tool to do it, but so far I didn't find it very reliable (causes random segfaults). I would say for now hardcoded is better but in a future we can potentially improve on this.

Reply:
ok